app/services/decisions/outgoings_decision_tree.rb

```diff
@@ -17,5 +19,15 @@ def destination # rubocop:disable Metrics/MethodLength
         raise InvalidStep, "Invalid step '#{step_name}'"
       end
     end
+
+    private
+
+    def after_housing_payment_type
+      if form_object.housing_payment_type.value == :board_and_lodging
+        edit(:board_and_lodgings)
+      else
+        edit(:council_tax)
```

Comment:
The `board_and_lodging` and `board_and_lodgings` is causing a lot of confusion - can the route name alias `board_and_lodgings` be the same as the type value `board_and_lodging` or is that too difficult to alter?